tests/baselines/reference/functionWithUseStrictAndSimpleParameterList_es2016.errors.txt

```diff
@@ -0,0 +1,131 @@
+tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts(35,32): error TS1345: This parameter is not allowed with 'use strict' directive.
+tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts(36,5): error TS1346: 'use strict' directive cannot be used with non-simple parameter list.
+tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts(39,23): error TS1345: This parameter is not allowed with 'use strict' directive.
+tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts(39,31): error TS1345: This parameter is not allowed with 'use strict' directive.
+tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts(41,5): error TS1346: 'use strict' directive cannot be used with non-simple parameter list.
+
+
+==== tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts (19 errors) ====
+    function a(a = 10) {
+               ~~~~~~
+!!! error TS1345: This parameter is not allowed with 'use strict' directive.
+!!! related TS1348 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:2:5: Use strict directive used here.
+        "use strict";
+        ~~~~~~~~~~~~~
+!!! error TS1346: 'use strict' directive cannot be used with non-simple parameter list.
+!!! related TS1347 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:1:12: Non-simple parameter declared here.
+    }
+    
+    export var foo = 10;
+    function b(a = 10) {
+    }
+    
+    function container() {
+        "use strict";
+        function f(a = 10) {
+        }
+    }
+    
+    function rest(...args: any[]) {
+                  ~~~~~~~~~~~~~~
+!!! error TS1345: This parameter is not allowed with 'use strict' directive.
+!!! related TS1348 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:16:5: Use strict directive used here.
+        'use strict';
+        ~~~~~~~~~~~~~
+!!! error TS1346: 'use strict' directive cannot be used with non-simple parameter list.
+!!! related TS1347 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:15:15: Non-simple parameter declared here.
+    }
+    
+    function rest1(a = 1, ...args) {
+                   ~~~~~
+!!! error TS1345: This parameter is not allowed with 'use strict' directive.
+!!! related TS1348 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:20:5: Use strict directive used here.
+                          ~~~~~~~
+!!! error TS1345: This parameter is not allowed with 'use strict' directive.
+!!! related TS1348 tests/cases/conformance/functions/functionWithUseStrictAndSimpleParameterList_es2016.ts:20:5: Use strict directive used here.

```

Comment:
Nit: Message should be `'use strict' directive` and not `Use strict directive` to match the other diagnostic.